moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -189,26 +191,30 @@ class CurrentStateMonitor
   }
 
 private:
-  void jointStateCallback(const sensor_msgs::JointStateConstPtr& joint_state);
+  void jointStateCallback(const sensor_msgs::msg::JointState::SharedPtr joint_state);
   void tfCallback();
 
-  ros::NodeHandle nh_;
+  // ros::NodeHandle nh_;
+  rclcpp::Node::SharedPtr node;
```

Comment:
There is `node` and there is `node_`. Also, we should keep the order of fields and functions the same to facilitate review and syncs